app/controllers/application_controller.rb

```diff
@@ -1,3 +1,7 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
+
+  def aplication
```

Comment:
__[staging][Misspell]__

"aplication" is a misspelling of "application"


<!-- Comment by [Sider](https://sider.review) -->